src/test/java/uk/gov/dhsc/htbhf/steps/AddressLookupSteps.java

```diff
@@ -4,6 +4,7 @@
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 import org.openqa.selenium.WebElement;
+import uk.gov.dhsc.htbhf.page.NationalInsuranceNumberPage;
```

Comment:
I'm puzzled why this import was added by your PR?

Reply:
Removed the unused import.